main.py

```diff
@@ -140,6 +142,10 @@ def main():
     # keyword_matches = []  # unused list?
 
     for medium in plex.media:
```

Comment:
@alex-phillips How does this look?

Reply:
Just about there! Instead of adding in a new initialization of `Plex`, you should just include the conditional logic above around like 120ish where we are already checking for `args.library`. Just add in `args.library or PLEX_LIBRARY` at that location.